staging/src/k8s.io/apiserver/pkg/storage/cacher/ready.go

```diff
@@ -46,6 +48,7 @@ type ready struct {
 }
 
 func newReady() *ready {
+	metrics.RecordWatchCacheReady(false)
```

Comment:
I think I would add this metric somewhere here https://github.com/openshift/kubernetes/blob/c918fbd4295af4b8d0fd6c6b9d14172f069d6890/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L464


and for the off scenario somewhere here https://github.com/openshift/kubernetes/blob/c918fbd4295af4b8d0fd6c6b9d14172f069d6890/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L468

We could slightly change WatchCacheInitializations to be a gauge or depreciate it and create a new one. I think the latter is better.